README.md

```diff
@@ -189,6 +189,16 @@ String archiveId = archive.getId();
 The `Archive.OutputMode.COMPOSED` setting (the default) causes all streams in the archive to be
 recorded to a single (composed) file.
 
+You can set the composed archive resolution to either "640x480" (SD, the default) or "1280x720" (HD) using the ArchiveProperties builder.
+Any other value will result in an exception.The property only applies to composed archives. 
+If you set this property and also set the outputMode property to "individual", the method results in an `InvalidArgumentException`.
+
+```java
+import com.opentok.ArchiveProperties;
+
+ArchiveProperties properties = new ArchiveProperties.Builder().resolution("1280x720").build();
+```
```

Comment:
I don't see the `startArchive` method implementation with the resolution in this PR. Is this here by accident?

Reply:
This is overlap of https://github.com/opentok/Opentok-Java-SDK/pull/152/files readme description. I should have added it there, but it is here..